### ISSUE TITLE
Full Accessibility & 508 Office Audit eBenefits - Ch31

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#27996]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [ ] VFS: Using the Slack Workflow ([User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)), notify Collaboration Cycle team of this Github ticket in #vfs-platform-support. 
- [ ] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [ ] VFS: Close this ticket when all steps above are completed.

## Artifacts
- [x] Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
**Any test user should work**
- [x] Link to verification that accessibility checks are complete:
  - [x] (preferred) Link to your [TestRail Platform accessibility test plan](https://dsvavsp.testrail.io/index.php?/cases/view/1005&group_by=cases:section_id&group_order=asc&group_id=1067&display_deleted_cases=0).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
      - [x] Our [508 folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/teams/ebenefits/features/apply-vre-ch31/508)
- List of known issues (with links to tickets when applicable)

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
FYI @joshkimux @AngelaFowler82  - as always, please let us know if you need help from Platform on this. thanks!

**Comment by AngelaFowler82** [comment]
Hello,

Could you please include the link to the component in staging as well as the scope of the review? As this ticket is currently written I have no idea what I’m supposed to be auditing.

Thank you kindly.

**Comment by jason-gcio** [comment]
> Hello, Could you please include the link to the component in staging as well as the scope of the review? As this ticket is currently written I have no idea what I’m supposed to be auditing. Thank you kindly.

You bet: https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start

**Comment by joshkimux** [comment]
@jason-gcio Hi Jason! Thank you for getting this full audit started up! 

It looks like there are still tickets in staging that need to be resolved-- a11y specialists will hold off from conducting the full audit until those are closed out. Let me know if it would be easier if I transition all those tickets to this new ticket 🙇
